Classes/People/PeopleViewController.swift

```diff
@@ -30,6 +30,8 @@ PeopleSectionControllerDelegate {
     private var owner: String
     private var repo: String
 
+    var wasDismissedByDone: Bool = false
```

Comment:
Should this variable be included in the `delegate` maybe? What do you think?

Reply:
Are you talking about `ContextMenuDelegate`? Maybe we can modify its functions signatures to add this flag:

```
public protocol ContextMenuDelegate: class {
    func contextMenuWillDismiss(viewController: UIViewController, animated: Bool, doneTapped: Bool)
    func contextMenuDidDismiss(viewController: UIViewController, animated: Bool, doneTapped: Bool)
}
```

Reply:
Yes, that sounds good!